File: SocialSoft-Front/Models/PlanSuscripcion.cs
using Newtonsoft.Json;

namespace SocialSoft.Client.Models
{
    /// <summary>
    /// Modelo que representa un Plan de Suscripción del sistema SocialSoft.
    /// Mapea directamente al modelo Java: pe.edu.practica.model.PlanSuscripcion
    /// </summary>
    public class PlanSuscripcion
    {
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("nombre")]
        public string Nombre { get; set; }

        [JsonProperty("costoMensual")]
        public double CostoMensual { get; set; }
    }
}

File: SocialSoft-Back/SocialSoft-Model/src/main/java/pe/edu/practica/model/PlanSuscripcion.java
package pe.edu.practica.model;

public class PlanSuscripcion {
    private int id;
    private String nombre;
    private double costoMensual;

    public PlanSuscripcion() {
    }

    public PlanSuscripcion(int id, String nombre, double costoMensual) {
        this.id = id;
        this.nombre = nombre;
        this.costoMensual = costoMensual;
    }

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public double getCostoMensual() { return costoMensual; }
    public void setCostoMensual(double costoMensual) { this.costoMensual = costoMensual; }
}